src/3rd_party_repos.c

```diff
@@ -615,9 +607,33 @@ enum swupd_code third_party_remove_binary(struct file *file)
 	return ret_code;
 }
 
+enum swupd_code third_party_bin_directory_exist(void)
```

Comment:
I would keep this function static

Reply:
good idea, this function is no longer being used anywhere outside that file